charts/telepresence/templates/deployment.yaml

```
@@ -286,6 +286,9 @@ spec:
           {{- template "traffic-manager-mounts" . }}
         {{- end }}
       {{- end }}
+      {{- with .schedulerName }}
+      schedulerName: {{ .schedulerName }}
```

```suggestion
      schedulerName: {{ . }}
```
unless you want `.schedulerName.schedulerName`.